packages/express/index.js

```diff
@@ -1,27 +1,22 @@
 'use strict';
 
-module.exports = {
-  get uri() {
-    return require('./lib/uri');
-  },
-  runServer(mode, options = {}) {
-    const core = require('@untool/core').bootstrap();
-    core.handleArguments(options);
-    return core.runServer(mode);
+const { bootstrap } = require('@untool/core');
```

Comment:
Just a tiny remark, but how about writing this code (in this file and all other files that export stuff) in a static manner, that doesn't mutate objects. I.e. like this:
```javascript
const { bootstrap } = require('@untool/core');
const uri = require('./lib/uri');

const configure = (config, options) => {
  return { ... };
};

const { runServer, createServer, createRenderer, renderLocations } = configure();

module.exports = exports = { runServer, createServer, createRenderer, renderLocations, configure };
```

With this approach it is easier to see what exactly gets exported and also this is a lot closer to ES modules (once they're implemented in Node):
```javascript
import { bootstrap } from '@untool/core';
import uri from './lib/uri';

const configure = (config, options) => {
  return { ... };
};

const { runServer, createServer, createRenderer, renderLocations } = configure();

export { runServer, createServer, createRenderer, renderLocations, configure };
```

Reply:
PTAL